CMSgov/design-system
# [WNMGS-919] Adding Medicare focus styles

Medicare is inheriting the focus styles from the core design system!

- Removed :focus styles that are no longer needed because they were duplicative of the SCSS set up in core


## How to test
Run the M.gov doc site locally with `yarn start:medicare` and see that the eggplant color is used
